build/components/setup.tex

```diff
@@ -212,45 +208,8 @@
 }
 \makeatother
 
-
-% ACRO SETUP
-\acsetup{
-    list/heading = section*,
-    list/name = {Abkürzungsverzeichnis},
-    list/template = description,
-    make-links = true,
-    link-only-first = false
-}
-
-% Standard Abkürzungsverzeichnis überschreiben -> einheitliche Einrückung
-\RenewAcroTemplate[list]{description}{%
-    \acronymsmapT{%
-        \AcroAddRow{%
-            \textbf{\acrowrite{short}}%
-            &
-            \acrowrite{long}%
-            \acropages
-            {\acrotranslate{page}\nobreakspace}%
-            {\acrotranslate{pages}\nobreakspace}%
-            \vspace{10pt}
-            \tabularnewline
-        }%
-    }%
-    \acroheading
-    \acropreamble
-    \noindent
-    \begin{tabular}{@{}ll}
-        \AcronymTable
-    \end{tabular}
-}
-
-% AUTO REMOVE/INSERT Abkürzungsverzeichnis
-% Abkürzungsverzeichnis überschreibt \UseAcroTemplate für \ac
-% New Counter to count used acronyms:
-\newtotcounter{acro_num}
-\def\oldUseAcroTemplate{} \let\oldUseAcroTemplate=\UseAcroTemplate
-\def\UseAcroTemplate{\stepcounter{acro_num}\oldUseAcroTemplate}
-
+% Glossar Setup
+\makenoidxglossaries

```

Comment:
Die Zeilen passen das Glossar an (Short ist Bold und Description ist länger (aktuell sehr eingequetscht) (hab bisher keine Möglichkeit gefunden dass es den rechtlichen Platz einnimmt deswegen eine feste Zahl (0.65)))
```suggestion
% Glossar Setup
\renewcommand{\glsnamefont}[1]{\textbf{#1}}
\setlength{\glsdescwidth}{0.65\textwidth}
\makenoidxglossaries

```

Reply:
Ja, das habe ich auch gemerkt, scheint eine Eigenschaft vom Super-Style zu sein? 🤔

Ich haus das manuell in den Commit rein